public/app/plugins/panel/stat/StatPanel.tsx

```diff
@@ -37,6 +37,9 @@ export class StatPanel extends PureComponent<PanelProps<PanelOptions>> {
         count={count}
         sparkline={sparkline}
         colorMode={options.colorMode}
+        // To preserve previous behavior, if other Panels use BigValue but don't pass a value,
+        // we default `hasGradient` to true, i.e. the background gradient WILL display.
```

Comment:
then you don't have to worry about it here :)

Reply:
For sure; better! Updated.